tests/backends/base/test_base.py

```diff
@@ -60,6 +61,43 @@ def test_check_database_version_supported_with_none_as_database_version(self):
         with patch.object(connection.features, "minimum_database_version", None):
             connection.check_database_version_supported()
 
+    def test_release_memory_without_garbage_collection(self):
+        try:
+            # Disable automatic garbage collection. To have control over when
+            # garbage collection is performed. This is necessary to ensure that another
+            # that thread doesn't accidentally trigger it by simply executing code.
```

Comment:
I think it should be 
```suggestion
            # garbage collection is performed. This is necessary to ensure that another
            # thread doesn't accidentally trigger it by simply executing code.
```